src/sparseml/yolov8/trainers.py

```diff
@@ -581,7 +598,7 @@ def __init__(self, model="yolov8n.yaml", type="v8") -> None:
         elif self.ValidatorClass == SegmentationValidator:
             self.ValidatorClass = SparseSegmentationValidator
 
-    def _load(self, weights: str):
+    def _load(self, weights: str, task=None):
```

Comment:
Since we are modifying the _load function, we should check if we get the same accuracy when running validation on sparsezoo-published models.

Reply:
The `_load` function wasn't really changed apart from now taking an input task and using that if the yolov8 itself can't guess the task from the loaded model.  Do you have a suggested model that you'd want me to run?

Reply:
Any sparseml obtained yolov8 should do. You could use [model](zoo:cv/detection/yolov8-s/pytorch/ultralytics/coco/pruned65_quant-none). I don't expect to see an error, but erring on the side of caution.

Reply:
Ran validation on the following model: 
```
sparseml.ultralytics.val \
  --model zoo:cv/detection/yolov8-s/pytorch/ultralytics/coco/pruned65_quant-none \
  --data coco.yaml
```

The mAP@0.50 and mAP@0.5:0.95 match the values listed on sparsezoo and the predictions also make sense